scripts/train/train.py

```diff
@@ -503,10 +504,13 @@ def main(cfg: DictConfig) -> Trainer:
 
     # Callbacks
     callbacks: List[Callback] = [
-        build_callback(str(name), callback_cfg)
+        build_callback(str(name), callback_cfg, om.to_container(logged_cfg))
```

Comment:
I think you need to set `resolve=True` in the `to_container` call? having trouble finding the default. or is `logged_cfg` already resolved?

Reply:
`logged_cfg` is already resolved (ln 188) - needed the deep copy  of it to have the config parameters before they were popped